src/store/modules/ADempiere/form/VPOS/order.js

```diff
@@ -374,7 +377,8 @@ export default {
       return new Promise(resolve => {
         const pos = getters.getVPOS
         if (isEmptyValue(order)) resolve({})
-        if (order.document_status.value === 'CO') {
+        console.log(order.document_status.value)
```

Comment:
please remove this console.log

Reply:
remove console    https://github.com/solop-develop/frontend-core/pull/1804/commits/1197d3806a38ed49944926e0d1762c2b4c7c2afe